react/src/pages/MapProject/MapProject.tsx

```diff
@@ -189,6 +196,15 @@ const LoadedMapProject: React.FC<LoadedMapProject> = ({
             featureCollection={featureCollection}
           />
         </div>
+        {selectedFeature && (
```

Comment:
```
{selectedFeature && (
  <div className={styles.detailContainer}>
    <AssetDetail
      selectedFeature={selectedFeature}
      onClose={() => setSelectedFeatureId(selectedFeature.id)} // Toggles the feature off
      isPublicView={activeProject.public} // Used to hide buttons like "Add Asset"
    />
  </div>
)}
```

general reactor suggestion that might be useful: if we switch the props of AssetDetail a bit, this might simplify AssetDetail:
*By passing in selectedFeature directly (and cause we only have AssetDetail if seletedFeature exists), you won't need internal checks in AssetDetail like `selectedFeature?.id`. 
selectedFeature: Passes the feature directly without needing internal checks.
* onClose: this can be used by the close button.

Then `isPublicView` is needed for hiding actions like the `Add asset from DesignSafe` button.